salt/modules/virt.py

```diff
@@ -4146,85 +4162,166 @@ def network_set_autostart(name, state='on', **kwargs):
         conn.close()
 
 
-def pool_define_build(name, **kwargs):
+def pool_define(name,
+                ptype,
+                target=None,
+                permissions=None,
+                source_devices=None,
+                source_dir=None,
+                source_adapter=None,
+                source_hosts=None,
+                source_auth=None,
+                source_name=None,
+                source_format=None,
+                transient=False,
+                start=True,  # pylint: disable=redefined-outer-name
+                **kwargs):
     '''
     Create libvirt pool.
 
     :param name: Pool name
-    :param ptype: Pool type
-    :param target: Pool path target
-    :param source: Pool dev source
```

Comment:
I don't like the idea of removing source.

At the very least, we should go through a deprecation process of this, and allow all of the other `source_*` things to still be defined in source.

Reply:
cool, if it hasn't been released yet, then I am ok with it if @homolkad is.

Reply:
It is ok. Nice job.